packages/cashier/src/types/stores/root-store.types.ts

```diff
@@ -2,11 +2,14 @@ import { TClientStore } from './client-store.types';
 import { TCommonStore } from './common-store.types';
 import { TUiStore } from './ui-store.types';
 
-export type RootStore = {
+export type TRootStore = {
     client: TClientStore;
     common: TCommonStore;
     modules: any;
```

Comment:
for deposit-store, error-store and iframe-store can we have interface and then have the class inside store to implement from it, will that be possible, if that is the case then it will help us to remove modules:any once all the store is converted into ts

Reply:
@vinu-deriv Cool idea 👍🏻

We can do this:
```ts
export type TRootStore = {
    client: TClientStore;
    common: TCommonStore;
    modules: {
        cashier: CashierStore;
    };
    ui: TUiStore;
};
```

As far as I can see the only module in `modules` is the `cashier` store, Am I right?

Reply:
@vinu-deriv Thank you for the suggestion, Good catch 👌🏻

https://user-images.githubusercontent.com/72082844/187624491-12ea799e-683b-4b01-a631-f64944aa78bc.mov

Reply:
nice work @farzin-fs